_content/monographs/Kamma-and-its-fruit_nyanaponika.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Kamma and Its Fruit"
+editor:
+  - nyanaponika
```

Comment:
editor is not a list! Just `editor: slug` 😊

Also: works can have up to five authors, so can we just add the five authors from this collection?

```yaml
authors:
  - nyanaponika
  - "Francis Story"
  - "Bhikkhu Ñāṇajīvako"
  - gorkom
  - "Leonard A. Bullen"
```

In which case, no need to mark Nyanaponika as editor, since he'll be an author already.

Reply:
Got it. Thanks! :)